ingestors/kafka/src/main/scala/hydra/kafka/algebras/KafkaClientAlgebra.scala

```diff
@@ -289,12 +311,12 @@ object KafkaClientAlgebra {
       }
 
       private def consumeMessages[A](
-                                      keyDeserializer: Deserializer[F, A],
+                                      keyDeserializer: Deserializer[F, Either[Throwable, A]],
```

Comment:
We need this to only skip for our internal topics. We don't want it to skip for any external topics.